Aptoide/aptoide-client-v8
# User unable to comment store after login

What does this pull request do?

This pull request fixes an issues regarding with user login validation when replying to a comment inside comment list.


How should this be manually tested?

1. login inside app
2. go to stores tab
3. open store
4. open "more" inside comments group to open comment list
5. check if reply button shows the reply popup (should work)
6. go back x2
7. logout
8. open the same store as previously (inside store tab)
9. open "more" inside comments group to open comment list
10. press reply
11. using Snackbar, login in app
12. go back to comment list
13. press reply and a reply popup should happear. not the login snackbar again.


Tickets related to this pull-request: 

- [ ] [AN-1083](https://aptoide.atlassian.net/browse/AN-1083)
